lib/trento/clusters.ex

```diff
@@ -176,6 +180,51 @@ defmodule Trento.Clusters do
     end
   end
 
+  @spec get_cluster_hosts(String.t()) :: [HostReadModel.t()]
+  def get_cluster_hosts(cluster_id) do
+    query =
+      from c in ClusterReadModel,
+        where: c.id == ^cluster_id,
+        preload: [:hosts]
+
+    case Repo.one(query) do
+      nil -> []
+      cluster -> Map.get(cluster, :hosts, [])
+    end
+  end
+
+  @spec request_operation(atom(), String.t(), map()) :: {:ok, String.t()} | {:error, any}
+  def request_operation(operation, cluster_id, params)
+      when operation in ClusterOperations.values() do
+    operation_id = UUID.uuid4()
+
+    # Design decision: in order to keep things simpler, the first node is selected as designated
```

Comment:
questions to better understand:

- `is_dc` is meant to be used in the operation specification in wanda, specifically in the predicate, correct?
So that wanda can filter out from the target list what is a non DC.
- an alternative could be requesting the operation for only one target (ie only one node of the cluster).

In any case I am fine with delegating that filtering to wanda.

Reply:
Yes, it will be for that matter. I just wanted to have only `request_operation` for cluster, where we send all the targets. This way we don't need to touch this function again, and all cluster operations would be covered.
This means using the `is_dc` attribute in wanda catalog of course